python/sglang/lang/backend/vertexai.py

```diff
@@ -50,13 +52,37 @@ def generate(
                 if s.cur_images
                 else s.text_
             )
+
+        debug_request_id = str(uuid.uuid4())
+        s.log_debug(
```

Comment:
It is better to add `s.log_debug` to `interpreter.py` so we do not need to do it for every backend file.